Standards/scs-0216-v1-high-security-setup.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Kubernetes high security setup
+type: Standard
+status: Draft
+track: KaaS
+---
+
+## Introduction
+
+A Kubernetes instance is provided as a cluster, which consists of a set of machines,
+so called nodes. These clusters are configured depending on the needs of its users, which
+is possible due to the flexibility of Kubernetes.
+Due to the regular changes, there are always new security features to deploy and use in Kubernetes.
+Nevertheless, a provider (or even a customer) needs to take action in order to achieve a
+hardened, secure cluster due to the myriad of configurations possible. This is especially
+the case since Kubernetes ships with insecure features and configurations out of the box,
+which will need to be mitigated by an administrator with the proper knowledge.
+Hardened Kubernetes clusters are desirable regardless of the possible thread model, since
+higher security doesn't necessarily mean higher complexity in this case.
+
+## Motivation
+
+Kubernetes clusters are highly configurable to fit a wide array of use-cases and setups.
+This makes them flexible and scalable to a certain extent, which is probably also why they
+found such a wide adoption. But its also obvious, that security problems can arise if a
+cluster isn't configured properly to mitigate them.
+These security risks can potentially be exposed in many different parts of a cluster, e.g.
+different APIs, authorization and authentication procedures or even Pod privilege mechanisms.
+In order to mitigate these problems, different hardening and prevention steps and mechanisms
+could be used to increase the security of a Kubernetes setup.
```

Comment:
The Motivation section duplicates the introduction to a certain degree, which means we have an opportunity to make the text shorter. We should use such opportunities ...

Reply:
Updated the `Motivation` section a little bit and shortened the text.

Reply:
Resolving this, since its open for over 2 months now.